### DISCUSSION TITLE
neovim feedback: installation issue

### DISCUSSION THREAD

**Discussion by dgdosen** [community/community#6847] [Copilot Conversations] [answered]
I installed neovim via homebrew `brew install neovim --HEAD`
I installed copilot.vim via vim-plug

If I try installing the plugin to neovim (using `:PlugInstall`)

I get this error:

`
x copilot.vim:
    fatal: invalid reference: master
`
I was able to run `:Copilot setup` - and authenticated,

When I then try to edit a file (say typescript or ruby) I get an error:

```
Error detected while processing function <SNR>195_Trigger[10]..copliot#Complete[7]..copiloit#doc#Get[11]..copilot#doc#UTF16Width:'
E945 Range too large for character clas
```
I also tried installing (then removed) the plugin to .config - but get the same runtime error.

Any advice?

**Comment by tpope** [comment, marked as answer]
~~Try `:set regexpengine=0`. (This is no longer necessary in the latest release.)~~

The `:PlugInstall` is failing because vim-plug itself needs to be updated.

**Reply by dgdosen** [reply]
That worked! Thanks. Although there's still an error during install:
`x copilot.vim: fatal: invalid reference: master`

**Reply by tpope** [reply]
What `Plug` line did you add to your init.vim?

**Reply by dgdosen** [reply]
`Plug 'github/copilot.vim'`

I get the same error when running `:PlugUpdate`

**Reply by tpope** [reply]
You can try adding `Plug 'github.com', {'branch': 'release'}`. But it should already be defaulting to that.

**Reply by tpope** [reply]
Oh you should try updating vim-plug. It used to erroneously assume `master`.

https://github.com/junegunn/vim-plug/commit/4a3c5e7ac280b15d7279ef48e6a915cf49023cc6

**Reply by dgdosen** [reply]
That worked... I hadn't updated vim-plug since early 2020...

**Reply by mike-clark-8192** [reply]
tpo.pe: can you add the bit about updating vim-plug to your main comment?
